homeassistant/components/system_bridge/sensor.py

```diff
@@ -283,15 +280,17 @@ async def async_setup_entry(
                 SystemBridgeSensor(coordinator, description, entry.data[CONF_PORT])
             )
 
-    displays = []
-    for display in coordinator.data.display.displays:
-        displays.append(
+    displays: list = []
```

Comment:
Specify the contents of the list (I'm not sure what the return type is, but, for the example, let's assume it is a string)

```py
    displays: list[str] = []
```